CMakeLists.txt

```diff
@@ -275,9 +275,13 @@ if (SINGULARITY_USE_HDF5)
   # hdf5, for data interopability
   # type: external
   enable_language(C)
+  set(_hdf5_comp "C;HL")
+  if(SINGULARITY_USE_FORTRAN)
+    set(_hdf5_comp "${_hdf5_comp};Fortran")
```

Comment:
I don't think we require fortran hdf5 interface at all do we?

Reply:
Correct; I thought this fixed an issue downstream but I was likely incorrect. Let me sort that out first and I'll remove this once I'm sure it's erroneous

Reply:
Ooohhhh yes this is all coming back to me. If its needed even for silly reasons we can leave it in for now. 

@Yurlungur the context is this may be needed due to hdf5/cmake combo is buggy and the fortran is needed in EAP.

Reply:
ok I see.